apps/website-25/src/lib/api/db/tables.ts

```diff
@@ -1,14 +1,17 @@
 import { Item, Table } from 'airtable-ts';
 
 export interface Course extends Item {
-  id: string,
-  title: string,
-  description: string,
-  displayOnCourseHubIndex: boolean,
   certificationBadgeImage: string,
   certificatonDescription: string,
-  path: string,
+  description: string,
   detailsUrl: string,
+  displayOnCourseHubIndex: boolean,
```

Comment:
```suggestion
  displayOnCourseHub: boolean,
```
Not introduced in this PR, but I first read it as meaning this defines the order it should display in

Reply:
Possibly `isDisplayedOnCourseHub` or `isDisplayedOnCourseHubIndex` would be clearer